tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -8200,31 +8274,51 @@ def globalWriteElements(self, kernel, tPA, tPB, vectorWidths, elements,
         betas = [False, True] if hasBeta else [False]
       if edges is None:
         edges = [False, True] if self.do["EdgeWrite"] else [False]
+      if biasDims is None:
+        biasDims = [0, 1] if self.states.BiasDim else [0]
       writeLabels = {}
       for beta in betas:
         writeLabels[beta] = {}
         for edge in edges:
           writeLabels[beta]["EdgeCheck0"] = Label(self.labels.getNameInc("GW_B%u_E%u_EdgeCheck0" % ( 1 if beta else 0, 1 if edge else 0) ), "")
           writeLabels[beta]["EdgeCheck1"] = Label(self.labels.getNameInc("GW_B%u_E%u_EdgeCheck1" % ( 1 if beta else 0, 1 if edge else 0) ), "")
-          writeLabels[beta][edge] = Label(self.labels.getNameInc("GW_B%u_E%u" % ( 1 if beta else 0, 1 if edge else 0) ), "")
+          writeLabels[beta][edge] = {}
+          for biasDim in biasDims:
+            writeLabels[beta][edge][biasDim] = Label(self.labels.getNameInc("GW_B%u_E%u_BD%u" % ( 1 if beta else 0, 1 if edge else 0, biasDim) ), "")
```

Comment:
It seems we don't need postfix "_BD"  for BiasDim == 0 kernel

Reply:
After using UseBias=[0,1,2,3] to control the bias direction. Now only when UseBias=3, bias support both MN directions, will add the postfix "_BD"

new code at here:
https://github.com/ROCm/hipBLASLt/pull/566/commits/be56fdad4516558712b565e75fba39ba92b15f05#diff-e47859bdb1274fe2c6254b728f1652f2001127828c4aeb41e8f4bff236c9c2a2R8288